src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1075,11 +1075,10 @@ private static NamedTypeSymbol GetDelegateType(BoundExpression expr)
                 methods = constructedMethods.ToImmutableAndFree();
             }
 
-            if (methods.Length == 1 && !extensionMethodsOfSameViabilityAreAvailable)
+            if (methods.Length == 1 && methods[0].Arity == 0 && !extensionMethodsOfSameViabilityAreAvailable)
```

Comment:
Could you please add a test for a case with explicit type arguments? Also please capture all explanations from this thread as comment in the source. They will be very useful.  

---
In reply to: [67028724](https://github.com/dotnet/roslyn/pull/11981#discussion_r67028724) [](ancestors = 67028724,67027973,67027377,67019197)